docs/extensions/readme.md

```diff
@@ -40,43 +34,13 @@ Those extensions are defined as additional description of the RestAPI, and do no
 - [x-ms-request-id](#x-ms-request-id) - allows to overwrite the request id header name
 - [x-ms-client-request-id](#x-ms-client-request-id) - allows to overwrite the client request id header name
 
-# Metadata Extensions
-
-## x-ms-secret
-
-This extension is used to annotate the secret property. The value type is boolean and the allowed value is either true or false. Secrets should never expose on a GET. If a secret does need to be returned after the fact, a POST api can be used to allow for granular RBAC.
- 
-**Rule**
-
-* When applying this extensions as: "x-ms-secret": true; means that this property must not exist in response.
-* When applying this extensions as: "x-ms-secret": false, it has same effect as not applying it.
-* When a property is modeled as both "required" and "x-ms-secret": true, which means that this property must not exist in the response but has to be present in request.
-* "x-ms-secret": true is equivalent to "x-ms-mutability": ["create", "update"].
-* When a property is modeled as "x-ms-mutability": ["read"], then applying this extension as "x-ms-secret": true is not allowed.
-* When a property is modeled as "readonly": true, then applying this extension as "x-ms-secret": true is not allowed.
+### Metadata extensions
 
-**Schema**:
-`true|false`
+Those extensions are defined as additional description of the RestAPI, and do not trigger at the moment any changes in code generation.
 
-**Example**:
+- [x-ms-secret](#x-ms-secret) - This extension is used to annotate the secret property.
 
-```json5
-"SecretAuthInfo": {
-      "type": "object",
-      "description": "The authentication info when authType is secret",
-      "properties": {
-        "name": {
-          "description": "Username or account name for secret auth.",
-          "type": "string"
-        },
-        "secret": {
-          "description": "Password or account key for secret auth.",
-          "type": "string",
-          "x-ms-secret": true
-        }
-    }
-}
-```
+- [x-ms-identifiers](#x-ms-identifiers) - indicates the identifying properties of objects in the array
```

Comment:
Is this extension already implemented by downstream system? Otherwise, it will be better to call out `InProgress` state.

Reply:
I'm talking with Arthur Ning on the planning. Yes, I can mark it as InProgress